editor/store/reducer.js

```diff
@@ -712,9 +712,9 @@ export function metaBoxes( state = defaultMetaBoxState, action ) {
 	}
 }
 
-export function browser( state = {}, action ) {
-	if ( action.type === 'BROWSER_RESIZE' ) {
-		return { width: action.width, height: action.height };
+export function mobile( state = false, action ) {
```

Comment:
minor: I would prefer if the state shape was something like state.browser.breakpoints.mobile or state.breakpoints.mobile, as it better identifies the meaning the values when seeing the state directly and leaves the door open for the addition of new breakpoints.

Reply:
I personally like the simplicity of the boolean state value. Let's keep it as is and revisit if we need more breakpoints.